src/main/webapp/resources/js/components/ant.design/ScrollableModal.tsx

```diff
@@ -3,26 +3,35 @@
  */
 
 import React from "react";
-import styled from "styled-components";
+import type { ModalProps } from "antd";
 import { Modal } from "antd";
+import styled from "styled-components";
 
-const ScrollBodyModal = styled(Modal)`
+const ScrollBodyModal = styled(Modal)<{ maxHeight: number }>`
   .ant-modal-body {
     padding: 15px 25px;
     overflow-y: auto;
     max-height: ${(props) => `${props.maxHeight}px` || `600px`};
   }
 `;
 
+type ScrollableModalProps = ModalProps & {
+  maxHeight?: number;
+  children: JSX.Element;
+};
+
 /**
  * React component to be used when you want a scrollable content in your ant design modal.
  *
- * @param {string} maxHeight - maximum height of the scrollable body
- * @param {element} children - the modal content
- * @returns {*}
- * @constructor
+ * @param maxHeight - maximum height of the scrollable body
+ * @param children - the modal content
+ * @param props - modal props
  */
-export function ScrollableModal({ maxHeight = 600, children, ...props }) {
+export function ScrollableModal({
+  maxHeight = 600,
+  children,
+  ...props
```

Comment:
Shouldn't props also have a type?

Reply:
No because that is included in the `M`odalProps & ... in the declaration on line 10